src/modules/0.0.14/aws_elb/sql/create_custom_constraints.sql

```diff
@@ -0,0 +1,14 @@
+create or replace function check_load_balancer_availability_zones(_load_balancer_name varchar, _availability_zones varchar[]) returns boolean
```

Comment:
can you add some comments on the file to explain the purpose of it? for me as a newcomer is still difficult to understand the purpose. This one is pretty straightforward but i am sure that there will be other functions that will not